src/module/module.py

```diff
@@ -20,8 +20,9 @@ def module_main():
     Implements module's main logic for inputting data.
     Function description should not be modified.
     """
-
+    
     log.debug("Inputting data...")
+    print(int(getenv('I2C_INTERFACE')[-1]))
```

Comment:
is this a debug print? if so remove it

Reply:
yes it is, removed in successive commits

Reply:
please remove such things prior to requesting a review

Reply:
yes, this is definitely true.